jtransc-rt/resources/hx/JA_L.hx

```diff
@@ -15,13 +15,20 @@ class JA_L extends JA_0 {
     }
 
 	{{ HAXE_METHOD_ANNOTATIONS }}
-    static public function fromArray(items:Array<Dynamic>, desc:String):JA_L {
+    static public function fromArray(desc:String, items:Array<Dynamic>):JA_L {
```

Comment:
Why did you switch the parameters?
This can always lead to bugs that are hard to debug.